central/networkpolicies/datastore/datastore_impl_test.go

```diff
@@ -229,6 +230,7 @@ func (s *netPolDataStoreTestSuite) TestAllowUpdateUndoNewer() {
 
 func (s *netPolDataStoreTestSuite) TestDisallowUpdateUndoOlder() {
 	oldTS := timestamp.TimestampNow()
+	time.Sleep(2 * time.Microsecond)
```

Comment:
Why do we need that?

Reply:
Testing locally, I have a flake. I suppose it comes from oldTS and newTS being equal.

Reply:
Then just mock the values and don't sleep
```
	oldTS, _ := timestamp.TimestampProto(time.Time{})
	newTS, _ := timestamp.TimestampProto(time.Time{}.Add(1))
```